ledger-bridge.js

```diff
@@ -3,65 +3,118 @@ import 'babel-polyfill'
 
 require('buffer')
 
-import TransportU2F from '@ledgerhq/hw-transport-u2f'
+import WebSocketTransport from "@ledgerhq/hw-transport-http/lib/WebSocketTransport"
 import LedgerEth from '@ledgerhq/hw-app-eth'
 import { byContractAddress } from '@ledgerhq/hw-app-eth/erc20'
 
+const BRIDGE_URL = "ws://localhost:8435"
+const TRANSPORT_CHECK_LIMIT = 10;
```

Comment:
This is the number of seconds we check for Ledger Live connectivity.  We should probably up this to like 30.